jwst/refpix/tests/test_clobber_ref.py

```diff
@@ -74,7 +74,8 @@ def test_decode_mask():
                [0, 3],
                [0, 7, 15]]
     for row in range(nrows):
-        bits = decode_mask(output[row], mask[row])
+        bits = decode_mask(mask[row])
         check[row] = (bits == compare[row])
 
+    # fmt: on
```

Comment:
could this be moved to be just after the `compare` array?

Reply:
I could, but then black was trying to remove the parentheses from the statement
`check[row] = (bits == compare[row])`
While the operator precedence does indeed make the parentheses unnecessary, I feel that for human readability they should be left in